src/main/java/org/mule/extensions/java/internal/parameters/ExecutableIdentifier.java

```diff
@@ -58,6 +62,27 @@ public String getElementName() {
     return match.matches() ? match.group(1) : "";
   }
 
+  /**
+   * Provides the type name of the {@link Executable} arguments being represented by {@code this} {@link ExecutableIdentifierFactory}.
+   * <p>
+   * For example, for method {@code public void echo(String message, int times)}, the result of invoking this method will be
+   * {@code ["String", "int"]}.
+   *
+   * @return the name of the {@link Executable} element being represented by {@code this} {@link ExecutableIdentifierFactory}.
```

Comment:
what's the difference between Optional.empty and empty Array?

Reply:
also, this is not returning the name of the element, but the name of its arguments